recipes/mkpi

```diff
@@ -44,3 +44,8 @@ fi
 if [ -n "$LOCK_PI" ]; then
   overcast run "$run_on" "sudo usermod --lock pi" "$@"
 fi
+
+if [ -n "$INSTALL_BLUEZ" ]; then
```

Comment:
@pgroudas This should be before the command to lock the `pi` user account, otherwise it will fail if using password-based authentication without first authorizing a SSH key.